types/tsconfig.json

```diff
@@ -8,6 +8,7 @@
     "types": [],
     "noEmit": true,
     "baseUrl": ".",
+    "skipLibCheck": true,
```

Comment:
this was probably for debugging purposes I'm imagining?

Reply:
it was, but it's also the right setting :P